src/components/screens/voting/index.test.js

```diff
@@ -46,16 +52,35 @@ describe('Delegates', () => {
   };
 
   it.skip('should allow to enable and disable voting mode', () => {
-    const wrapper = mountWithProps(Delegates, defaultProps, mockStore);
+    const wrapper = mountWithRouter(Delegates, defaultProps, mockStore);
     wrapper.find('.start-voting-button').at(0).simulate('click');
     expect(wrapper.find('.addedVotes')).to.have.lengthOf(1);
 
     wrapper.find('.cancel-voting-button').at(0).simulate('click');
     expect(wrapper.find('.addedVotes')).to.have.lengthOf(0);
   });
 
+  it.skip('should not be in edit mode', () => {
+    const wrapper = mountWithRouter(
+      Delegates,
+      defaultProps,
+      {
+        ...mockStore,
+        history: {
+          ...mockStore.history,
+          location: {
+            search: '?modal=votingSummary&isSubmitted=true',
+          },
+        },
+      },
+    );
+
+    expect(wrapper.find('.start-voting-button')).to.have.lengthOf(1);
```

Comment:
You can use `expect(wrapper).toContainMatchingElement(/matcher/)` instead

Reply:
For doing so we should refactor the whole unit test file from using expect from chai to the default expect. I think that would be contra-productive to do it now. What do you think?

Reply:
I checked the functionality. It is working as expected. I will approve once @UsamaHameed approves the code review